docs/src/markdown/usage.md

```diff
@@ -575,7 +571,7 @@ You can read more about the backrefs' features in the [Backrefs' documentation][
 
 It is not always clear when Package Control updates dependencies.  So to force dependency updates, you can run Package Control's `Satisfy Dependencies` command which will update to the latest release.
 
-### Using Backrefs in RegReplace Plugin
+### Using Backrefs in Regreplace Plugins
```

Comment:
RegReplace is the correct casing of the plugin name.

Reply:
Sorry for that, the title capitalization tools should have preserved manually introduced capitalizations, but apparently it didn't. *Mea culpa* for not checking.